xml/operations-cloudadmin_cli.xml

```diff
@@ -155,7 +155,7 @@
    Example output:
   </para>
 <screen># This indicates that the admin user is assigned the admin role within the default domain
-stack@hLinux:~$ openstack role assignment list --user admin --domain default
+ardana@sles:~$ openstack role assignment list --user admin --domain default
```

Comment:
I see why you'd want to change these prompts, but this is not really the right way. Take a look at entity-decl.ent, line 20-23 -- this is how we usually do prompts:

* We use the prompt tag which in HTML gets CSS to make it impossible to accidentally copy it (also adds gray [=user] or red [= root] color)
* We do not include the path, because that is often just very long.
* We do not usually include a machine name. (Except for setups where you need to be able to differentiate between machines.)
* We use the SUSE prompt separators (`>`/`#`).
* We normally use an entity for prompts.

Because `ardana` is kind of a special user, I would suggest to create a new `&prompt.ardana;` for user `ardana` and then use that everywhere (i.e. copy line 22 and change the entity name and user name in the prompt).